content/posts/networkx/GSoC-2022/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "GSoC 2022"
+date: 2022-06-09
+draft: false
+description: "Getting started"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["kpetridis24"]
+
+resources:
+  - name: featuredImage
+    src: "mjschwenne_GSoC.png"
```

Comment:
Please use a tool like `pngcrunsh` to compress the image. Looking at it, it should be easy to get well bellow 100kb.

Reply:
@tupui Thanks for the feedback. I added a significant ammount of information. Check it out and if there is more that can go into it, let me know.